packages/openapi-typescript/src/lib/redoc.ts

```diff
@@ -64,7 +65,7 @@ export async function parseSchema(schema: unknown, { absoluteRef, resolver }: Pa
     if (schema[0] === "{") {
       return {
         source: new Source(absoluteRef, schema, "application/json"),
-        parsed: JSON.parse(schema),
+        parsed: parseJson(schema),
```

Comment:
Just adds nicer syntax errors for better DX. In my benchmarks I can’t notice a significant performance difference between this and native `JSON.parse()`.